packages/libluajit/build.sh

```diff
@@ -2,79 +2,52 @@ TERMUX_PKG_HOMEPAGE=https://luajit.org/
 TERMUX_PKG_DESCRIPTION="Just-In-Time Compiler for Lua"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="1:2.1.1763148144"
+TERMUX_PKG_VERSION="1:2.1.1763148144+g5c8cee3"
 TERMUX_PKG_SRCURL=git+https://github.com/LuaJIT/LuaJIT.git
 TERMUX_PKG_GIT_BRANCH=v${TERMUX_PKG_VERSION:2:3}
-TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_BREAKS="libluajit-dev"
 TERMUX_PKG_REPLACES="libluajit-dev"
 TERMUX_PKG_EXTRA_MAKE_ARGS="amalg PREFIX=$TERMUX_PREFIX"
 TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_UPDATE_TAG_TYPE=newest-tag
 
 termux_pkg_auto_update() {
-	local current_version="${TERMUX_PKG_VERSION#*:}"
-	local response latest_version unix_timestamp_latest
-	local api_url='https://archlinux.org/packages/search/json/?name=luajit'
-	# Get the latest version from Arch Linux's API.
-	# Since this project doesn't do release tags,
-	# Repology doesn't return a meaningfully interpretable latest version.
-	# We should also identify ourselves via 'User-Agent' as a courtesy.
-	response="$(curl -s \
-		-H 'User-Agent: Termux update checker 1.0 (github.com/termux/termux-packages)' \
-		"${api_url}"
+	local latest_version current_version="${TERMUX_PKG_VERSION#*:}"
+	latest_version="$(
+		TERMUX_PKG_SRCURL="https://gitlab.archlinux.org/archlinux/packaging/packages/luajit" \
+		termux_gitlab_api_get_tag
 	)"
-	latest_version="$(jq -r '.results[0].pkgver' <<< "$response")"
-	unix_timestamp_latest="${latest_version##*.}"
-
-	if ! date -d "@${unix_timestamp_latest}" &> /dev/null; then
-		local summary
-		# shellcheck disable=SC2016
-		printf -v summary '%s\n' \
-			'### `libluajit`' \
-			''\
-			"Failed to get latest version of 'luajit'" \
-			"from '${api_url}'" \
-			''\
-			"Timestamp - $(date -d "@${unix_timestamp_latest}" --utc '+%Y-%m-%dT%H:%M:%SZ' 2>&1)" \
-			'Prettified `curl` response:' \
-			'```json' \
-			"$(jq -r <<< "$response")" \
-			'```'
-
-		if [[ -n "${GITHUB_STEP_SUMMARY:-}" ]]; then
-			# If this is the CI, output to the summary file
-			echo "$summary" >> "$GITHUB_STEP_SUMMARY"
-		else
-			# Otherwise output to whatever qualifies as FD2.
-			echo "$summary" >&2
-		fi
-		echo "WARN: Couldn't query new version. Staying at version '${TERMUX_PKG_VERSION}'."
-		return 0
-	fi
 
-	# If this isn't a dry-run add a human readable (ISO 8601)
-	# version of the timestamp as a comment to the version.
-	if [[ "${BUILD_PACKAGES}" != "false" && "$current_version" != "$latest_version" ]]; then
-		sed \
-			-e "s|^\(TERMUX_PKG_VERSION=.*\"\).*|\1 # $(date -d "@${unix_timestamp_latest}" --utc '+%Y-%m-%dT%H:%M:%SZ')|" \
-			-i "$TERMUX_PKG_BUILDER_DIR/build.sh"
-	fi
```

Comment:
Unless I get a really good idea for how to do this I'm just gonna drop the idea of adding a timestamp comment to the package version of LuaJIT, there's no good point to inject it before the `git push`.